internal/stack/certs.go

```diff
@@ -161,9 +180,16 @@ func verifyTLSCertificates(caFile, certFile, keyFile, name string) error {
 	options := x509.VerifyOptions{
 		Roots: certPool,
 	}
-	if name != "" {
-		options.DNSName = name
+	if service.Name != "" {
+		options.DNSName = service.Name
 	}
+
+	// By default ExtKeyUsageServerAuth is add to KeyUsages
+	// See https://github.com/golang/go/blob/master/src/crypto/x509/verify.go#L193-L195
```

Comment:
Good catch :+1: 

Looking to this I wonder if we also need to check the usages in https://github.com/elastic/elastic-package/pull/1646/files?diff=unified&w=1#diff-9b02d91c4a392a35e7a9ff3e09d1f61d6c7a9a2b42f7034a30b5feaf265ff15cR331

Reply:
Yeah I think this is the same right? We are adding https://github.com/elastic/elastic-package/pull/1646/files?diff=unified&w=1#diff-9b02d91c4a392a35e7a9ff3e09d1f61d6c7a9a2b42f7034a30b5feaf265ff15cR183

It is called ExtKeyUsage in template.